packages/manager/apps/dedicated/client/app/account/user/user.routes.js

```diff
@@ -28,6 +29,23 @@ export default /* @ngInject */ ($stateProvider) => {
           : Promise.resolve(null),
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('user_account'),
+
+      guides: /* @ngInject */ (currentUser) => {
+        Object.entries(GUIDES_LIST).forEach(([key, value]) => {
+          GUIDES_LIST[key].url =
+            typeof value.url === 'object'
+              ? value.url[currentUser.ovhSubsidiary] || value.url.DEFAULT
+              : value.url;
+        });
+        return GUIDES_LIST;
```

Comment:
I think we should return a new object instead of updating the existing one

Reply:
I changed my approach.